src/v2/view-builder/views/consent/EmailMagicLinkOTPTerminalView.js

```diff
@@ -1,19 +1,44 @@
 import { loc, View } from 'okta';
 import hbs from 'handlebars-inline-precompile';
 
+const generateGeolocationString = (location) => {
```

Comment:
style-nit: Consider adding a default function param to avoid the large `if (location)` block below.

Reply:
Updated!